api/swagger.yaml

```diff
@@ -3528,6 +3548,28 @@ components:
               items:
                 $ref: '#/components/schemas/vehicleType'
 
+    dangerousGoodsConstraint:
+      allOf:
+        - $ref: '#/components/schemas/constraint'
+        - properties:
+            description:
+              description: The description of a dangerous good. The exact meaning of the description depends on the `descriptionType` field.
+              type: string
+            descriptionType:
+              description: The type of description of the dangerous good.
+              type: string
+              enum:
+                - chemicalName
```

Comment:
Vanuit standaardisatie-oogpunt is het misschien nuttig om expliciet te vermelden dat de chemicalName ook een InCHi ( https://en.wikipedia.org/wiki/International_Chemical_Identifier ) of een CAS-number ( https://en.wikipedia.org/wiki/CAS_Registry_Number ) mag zijn.

Het voordeel van deze standaarden is dat bijv. de databases waar de brandweer veiligheidsinformatie uit haalt hier op geindexeerd zijn, terwijl er voor een ruwe "naam" tientallen varianten kunnen rondzweven voor dezelfde chemische stof.

Reply:
Je hebt overigens dit nieuwe constraint niet benoemd in de changelog bovenin.